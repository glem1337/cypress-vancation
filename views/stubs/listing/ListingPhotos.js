import { Col, Row } from 'antd';
import LayoutListing from './LayoutListing';

const ListingPhotos = () => (
  <LayoutListing withBackBtn>
    <Row>
      <Col span={24} lg={16}>
        <h1 className="text-headline mb-8">
          Add photos of your camper
        </h1>
        <p className="mb-24">
          Your camper photos are the most important part of your
          listing, we recommend uploading at least 5 photos to maximize bookings.
        </p>
      </Col>
      <Col span={24}>
        {/* TODO: To FRONT_END  here must be this component
           views/AddNewCamper/Photos/component.js
           */}
      </Col>
    </Row>
  </LayoutListing>
);

export default ListingPhotos;
