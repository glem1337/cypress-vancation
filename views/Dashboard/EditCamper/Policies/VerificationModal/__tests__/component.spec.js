import React from 'react';
import { shallow } from 'enzyme';

import ListingPhotos from '../component';

describe('VerificationModal component tests', () => {
  const props = {
    onReview: jest.fn(),
    onClose: jest.fn(),
  };

  const component = shallow(<ListingPhotos {...props} />);

  it('matches snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
