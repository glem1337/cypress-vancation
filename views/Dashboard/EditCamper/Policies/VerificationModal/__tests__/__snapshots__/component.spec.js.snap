// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VerificationModal component tests matches snapshot 1`] = `
<Modal
  className="main-modal main-modal--big"
  closable={true}
  closeIcon={
    <i
      className="icon icon-cross main-close-btn"
    />
  }
  footer={null}
  maskClosable={true}
  onClose={[MockFunction]}
  visible={true}
>
  <div
    className="main-modal__container"
  >
    <div
      className="d-flex pt-48 mb-48"
    >
      <i
        className="font-32 mr-16 icon icon-info in-yellow-1000"
      />
      <div
        className="mb-16 text-headline font-20 font-700"
      >
        <Memo(FormattedMessage)
          id="dashboard.editCamper.noticeDays.warning"
        />
      </div>
    </div>
    <div
      className="d-flex align-items-center justify-content-center"
    >
      <BtnGradient
        className="min-w-140 mr-16"
        isLoading={false}
        size="large"
        text={
          Object {
            "id": "shared.verifyMyLicence",
          }
        }
      />
      <Button
        onClick={[MockFunction]}
        size="large"
        type="link"
      >
        <Memo(FormattedMessage)
          id="dashboard.editCamper.verifyModal.verify.btn"
        />
      </Button>
    </div>
  </div>
</Modal>
`;
